tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -579,15 +652,64 @@ static uint8_t set_discoverable(const void *cmd, uint16_t cmd_len,
 
 	switch (cp->discoverable) {
 	case BTP_GAP_NON_DISCOVERABLE:
+		if (IS_ENABLED(CONFIG_BT_CLASSIC)) {
+			int err;
+
+			err = bt_br_set_discoverable(false, false);
+			if ((err < 0) && (err != -EALREADY)) {
+				return BTP_STATUS_FAILED;
+			}
+		}
+
 		ad_flags &= ~(BT_LE_AD_GENERAL | BT_LE_AD_LIMITED);
 		atomic_clear_bit(&current_settings, BTP_GAP_SETTINGS_DISCOVERABLE);
 		break;
 	case BTP_GAP_GENERAL_DISCOVERABLE:
+		if (IS_ENABLED(CONFIG_BT_CLASSIC)) {
+			int err;
+
+			err = bt_br_set_connectable(true);
+			if (err == -EALREADY) {
+				err = bt_br_set_discoverable(false, false);
+				if ((err < 0) && (err != -EALREADY)) {
+					return BTP_STATUS_FAILED;
+				}
+			} else if (err < 0) {
+				return BTP_STATUS_FAILED;
+			}
+
+			err = bt_br_set_discoverable(true, false);
+			if (err < 0) {
+				return BTP_STATUS_FAILED;
+			}
+			ad_flags &= ~BT_LE_AD_NO_BREDR;
```

Comment:
I assume that there is no option to disable BR/EDR on runtime?  (ie BTP_GAP_SETTINGS_BREDR is always set when BR/EDR is compiled in?)